Remarkable-Donkeys/bangazon
# Ticket 11: Displaying inactive customers

# Steps To Test #
1. fetch Tyler-11 branch
2. ensure that you have some customers with shopping carts and some without
3. try requesting api/customer/get?active=false

Ticket url: https://github.com/Remarkable-Donkeys/bangazon/issues/11

### Description of what should happen when steps are followed ###
If the test is successful you when requesting api/get?active=false you should get customers that do not have any shopping carts.

**At least one person needs to approve pull request before merged to master**
